iclr_wrap_up/utils.py

```diff
@@ -69,4 +69,33 @@ def is_dense_like(layer):
     Returns:
         True if layer has attribute 'kernel', False otherwise
     """
-    return hasattr(layer, 'kernel')
\ No newline at end of file
+    return hasattr(layer, 'kernel')
+
+
+def get_min_max(activations_summary, layer_number, neuron_number=None):
+    epochs_in_activation_summary = [int(epoch) for epoch in activations_summary]
+    epochs_in_activation_summary = np.asarray(sorted(epochs_in_activation_summary))
+
+    total_max = 0
+    total_min = 0
+    for epochs in epochs_in_activation_summary:
+        activations = activations_summary[f'{epochs}/activations']
+
+        if neuron_number is not None:
+            layer_activations = np.asarray(activations[str(layer_number)])
+            layer_activations = layer_activations.transpose()
+
+            current_max = np.max(layer_activations[neuron_number])
+            current_min = np.min(layer_activations[neuron_number])
+        else:
+            layer_activations = np.asarray(activations[str(layer_number)])
+
+            current_max = np.max(layer_activations)
+            current_min = np.min(layer_activations)
+
+        if current_max > total_max:
```

Comment:
This custom logic could be replaced by 
`total_max = np.max(total_max, current_max)`